teku/src/main/java/tech/pegasys/teku/AbstractNode.java

```diff
@@ -175,8 +175,7 @@ private void reportBellatrixMergeOverrides(final TekuConfiguration tekuConfig) {
       reportTerminalBlockHashOverride = true;
       reportTerminalBlockHashEpochOverride = true;
     } else {
-      final Eth2NetworkConfiguration defaultConfiguration =
-          Eth2NetworkConfiguration.builder().applyNetworkDefaults(maybeEth2Network.get()).build();
+      final Eth2NetworkConfiguration defaultConfiguration = tekuConfig.eth2NetworkConfiguration();
```

Comment:
this was interesting - we recreated spec here which was made evident when the log.info was added